aptos-move/framework/aptos-framework/sources/configs/Stake.move

```diff
@@ -248,45 +257,44 @@ module AptosFramework::Stake {
         // update validator info (so network address/public key change takes effect)
         let i = 0;
         let len = Vector::length(&validator_set.active_validators);
+        let active_validators = Vector::empty();
         while (i < len) {
             let old_validator_info = Vector::borrow_mut(&mut validator_set.active_validators, i);
-            *old_validator_info = generate_validator_info(old_validator_info.addr);
+            let stake_pool = borrow_global_mut<StakePool>(old_validator_info.addr);
+            stake_pool.voting_power = stake_pool.next_epoch_voting_power;
+            let new_validator_info = generate_validator_info(old_validator_info.addr);
+            if (new_validator_info.voting_power >= validator_set.minimum_stake &&
+                new_validator_info.voting_power <= validator_set.maximum_stake
```

Comment:
might it make sense to add a todo here to claim the maximum stake rather than just rejecting it?

Reply:
what does claim mean?